web/pdf_page_view.js

```diff
@@ -122,6 +122,12 @@ class PDFPageView {
     div.style.width = Math.floor(this.viewport.width) + "px";
     div.style.height = Math.floor(this.viewport.height) + "px";
     div.setAttribute("data-page-number", this.id);
+    div.setAttribute("role", "region");
+    this.l10n
+      .get("page_canvas", { page: this.id }, "Page {{page}}")
```

Comment:
Is it "correct" to re-use the existing `page_canvas` string here, since this isn't actually a canvas?
Furthermore, with these changes should we still keep setting https://github.com/mozilla/pdf.js/blob/24e598a895b712cbfa1f34deb12bb2c0f3575fee/web/pdf_page_view.js#L620-L622
or can that now become confusing to accessibility software with the *same* aria label being duplicated for *different* elements?

Finally, please remove the "Page {{page}}" part here since it's already handled by https://github.com/mozilla/pdf.js/blob/24e598a895b712cbfa1f34deb12bb2c0f3575fee/web/l10n_utils.js#L48

Reply:
Not really correct to re-use, but we'll end up re-triggering localization of it. I guess that's fine.

With the page div and canvas labeled the screen reader ends up reading it twice which isn't ideal.

Reply:
I thought i removed the page_canvas, must have missed that change...